api/src/config.ts

```
@@ -7,6 +7,8 @@ import { SnakeNamingStrategy } from './utils/snake-naming-strategy';
 const env = process.env;
 const getBoolOrDefault = (value: string, defaultValue: boolean) => (value ? value === 'true' : defaultValue);
 
+const getNumberOrDefault = (value: string, defaultValue: number) => (value ? parseInt(value, 10) : defaultValue);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)